Neos.Neos/Classes/Fusion/ConvertUrisImplementation.php

```diff
@@ -149,6 +150,14 @@ function ($matches) use ($externalLinkTarget, $resourceLinkTarget, $host) {
                 $target = null;
                 if ($externalLinkTarget !== '' && is_string($uriHost) && $uriHost !== $host) {
                     $target = $externalLinkTarget;
+
+                    if ($target === '_blank') {
+                        if (preg_match_all('~target="(.*?)~i', $linkText, $targetMatches)) {
+                            $linkText = preg_replace('/target="(.*?)"/', 'target="$1" rel="noopener"', $linkText);
+                        } else {
+                            $linkText = str_replace('<a', '<a rel="noopener"', $linkText);
+                        }
+                    }
```

Comment:
Hi @urbantrout, thanks for your contribution.

One question though. Why add this up here instead of further down?

Example:
    return str_replace('<a', sprintf('<a target="%s"', $target === '_blank' ? $target . '" rel="noopener' : $target), $linkText);

Reply:
Good point. Looks much cleaner this way. I will update the file.

Reply:
Thanks for pointing that out. I went with this solution:

`return preg_replace('/target=".*?"/', sprintf('target="%s"%s', $target, $target === '_blank' ? ' rel="noopener"' : ''), $linkText);`

Looks a bit cleaner to me ;)